task15/index.js

```diff
@@ -0,0 +1,94 @@
+const parent = document.getElementById("js-parent"); 
+const modalPlace = document.getElementById('js-modalPlace');
+
+const renderButtonForParent = () => {
+    const button = document.createElement('input');
+    modalPlace.style.display = "none"; 
+    button.id = "js-openBtn";
+    button.type = "button";
+    button.value = "click";
+    parent.appendChild(button);
+};
+renderButtonForParent();
+
+const openBtn = document.getElementById("js-openBtn");
+openBtn.addEventListener("click", (e) => {
+    modalPlace.className = "modal";
+    modalPlace.style.display = "block";
+    e.target.remove();
+    modalClose();
+});
+
+const modalClose = () => {
+    const closeBtn = document.getElementById("js-closeBtn");
+    closeBtn.addEventListener("click", () => {
+        requestValue();
+    });
+};
+
+const requestValue = () => {
+    const getName = document.getElementById("js-getName");
+    const getNumber = document.getElementById("js-getNumber");
```

Comment:
IMO
I felt that specifying the value here would make the code from line 32 more readable, what do you think?
```suggestion
    const getName = document.getElementById("js-getName").value;
    const getNumber = document.getElementById("js-getNumber").value;
```

Reply:
I can write like this! Amazing!
I learned a lot.
7dce551